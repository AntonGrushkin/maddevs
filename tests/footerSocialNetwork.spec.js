import {
  mount
} from '@vue/test-utils';
import FooterSocialNetwork from '@/components/Footer/footer-social-network';

describe('Footer social network', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(FooterSocialNetwork);
  });

  // ------ IMPORTANT ----- //
  test('is a Vue instance', () => {
    expect(wrapper.exists()).toBeTruthy();
  });

  test('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
  // --------------------- //
});
