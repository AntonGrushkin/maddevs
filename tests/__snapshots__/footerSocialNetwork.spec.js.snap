// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer social network renders correctly 1`] = `
<div
  class="footer-social-network"
>
  <a
    class="footer-social-network__social-newtork_link"
    href="https://github.com/maddevsio"
    target="_blank"
  >
    <span
      class="footer-social-network__icon footer-social-network__icon--github"
    />
  </a>
   
  <a
    class="footer-social-network__social-newtork_link"
    href="https://www.linkedin.com/company/mad-devs"
    target="_blank"
  >
    <span
      class="footer-social-network__icon footer-social-network__icon--linkedin"
    />
  </a>
   
  <a
    class="footer-social-network__social-newtork_link"
    href="https://www.facebook.com/maddevsllc"
    target="_blank"
  >
    <span
      class="footer-social-network__icon footer-social-network__icon--facebook"
    />
  </a>
   
  <a
    class="footer-social-network__social-newtork_link"
    href="https://www.instagram.com/maddevsio/"
    target="_blank"
  >
    <span
      class="footer-social-network__icon footer-social-network__icon--instagram"
    />
  </a>
   
  <a
    class="footer-social-network__social-newtork_link"
    href="https://blog.maddevs.io/"
    target="_blank"
  >
    <span
      class="footer-social-network__icon footer-social-network__icon--medium"
    />
  </a>
   
  <a
    class="footer-social-network__social-newtork_link"
    href="https://twitter.com/MadDevsIO"
    target="_blank"
  >
    <span
      class="footer-social-network__icon footer-social-network__icon--twitter"
    />
  </a>
   
  <a
    class="footer-social-network__social-newtork_link"
    href="https://www.behance.net/maddevs"
    target="_blank"
  >
    <span
      class="footer-social-network__icon footer-social-network__icon--behance"
    />
  </a>
</div>
`;
